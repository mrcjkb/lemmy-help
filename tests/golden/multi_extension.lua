---@class lz.n.PluginSpec: lz.n.PluginBase,lz.n.PluginSpecHandlers,lz.n.PluginHooks
---
--- The plugin name (not its main module), e.g. "sweetie.nvim"
---@field [1] string

local U = {}
return U
