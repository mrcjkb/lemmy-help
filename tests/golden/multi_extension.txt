                                                               *lz.n.PluginSpec*
lz.n.PluginSpec : lz.n.PluginBase, lz.n.PluginSpecHandlers, lz.n.PluginHooks

    Fields: ~
        {1}  (string)
                        The plugin name (not its main module), e.g. "sweetie.nvim"



